### PR TITLE
closes #91699

### LINKED ISSUE #91699: Allow to set background color for active panel title
Currently it is possible to change the border color for the active panel title using `panelTitle.activeBorder`, allow a similar setting for the background color, maybe `panelTitle.activeBackground`.

### PR DESCRIPTION
**Pull request by a5hk**
This PR fixes #91699

### CODE REVIEW

File: src/vs/workbench/common/theme.ts [hunk 1]
```diff
@@ -221,6 +221,12 @@ export const PANEL_BORDER = registerColor('panel.border', {
 	hc: contrastBorder
 }, nls.localize('panelBorder', "Panel border color to separate the panel from the editor. Panels are shown below the editor area and contain views like output and integrated terminal."));
 
+export const PANEL_ACTIVE_TITLE_BACKGROUND = registerColor('panelTitle.activeBackround', {
```
Comment (aeschli) [inline src/vs/workbench/common/theme.ts]: `g` mising in `activeBackground`

**Comment by aeschli** [comment]
@bpasero For you to decide if the new color makes sense...

**Comment by bpasero** [comment]
LGTM besides the comment already made around the missing `g`. Assigning to @isidorn to merge if you think this is OK, it touches the composite switcher component. 

I am not 100% sure the CSS rules are correct if I look at where the title foreground is used, but it seems to work.

**Comment by isidorn** [comment]
Thanks a lot for your PR, however that feature request is not yet even on the backlog and we are not accepting it. Reason being: I am not convinced this background color is needed and it might go against our styling. Thus I will not merge this PR.
fyi @sbatten @misolori if you like these changes feel free to merge in

**Comment by a5hk** [comment]
For me the main reason is that it enables me to remove the border separating panels from main editor window (that red line in the picture below) and still be able to easily find the position of the resize handle.
b

To me this background 
nb

is basically a shorter indicator (resize handle) like the yellow line in the following picture.
y

**Comment by isidorn** [comment]
Ok thanks for the explanation. I would still keep this PR open and potentialy merge it in the future.

**Comment by misolori** [comment]
I wouldn’t merge this in give that #91699 is a backlog candidate with no votes. It would go against the principle of candidates.

**Comment by isidorn** [comment]
Closing this as it got outadated and the feature is no longer active

**Pull request closed without merging**